src/os/filestore/FileStore.h

```diff
@@ -95,6 +95,7 @@ enum {
   l_filestore_apply_latency,
   l_filestore_queue_transaction_latency_avg,
   l_filestore_last,
+  l_filestore_sync_pause_max_lat,
```

Comment:
Yeah, you are right... but it is so weird it is not segment fault when I ran locally with vstart.sh, I could "perf dump" my new added. Mmm...

Any way, let me fix it.